client/src/components/commonComponents/Dropdown/index.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './index.css';
+
+const langs = [
+  {
+    id: 1,
+    language: 'Chinese',
+  },
+  {
+    id: 2,
+    language: 'Spanish',
+  },
+  {
+    id: 3,
+    language: 'English',
+  },
+  {
+    id: 4,
+    language: 'Hindi',
+  },
+  {
+    id: 5,
+    language: 'Russian',
+  },
+];
+
+const Dropdown = ({ label, name, languages = langs }) => {
+  return (
+    <div className="drop-down">
+      <label htmlFor="select">
+        {label}
+        <select name={name} id="select">
+          {languages.map(({ id, language }) => (
```

Comment:
how can I take the value from the dropdown  !! 
where is the handlechange(onChange) ???

Reply:
I added it it in the props with the value